services/bundle_analysis.py

```diff
@@ -40,6 +41,36 @@ def load_time_conversion(size):
     return round((8 * size) / (1024 * 1024 * 3), 1)
 
 
+@dataclass
+class BundleAnalysisReport(object):
+    def __init__(self, report: SharedBundleAnalysisReport):
+        self.report = report
+        self.report_bundles = []
+        self.total_size = 0
+        for bundle in self.report.bundle_reports():
```

Comment:
TODO as we spoke previously:

1) add functionality to calculate total BundleAnalysisReport size in shared
2) rethink how to ensure we close/start all db connections in shared as well, fine here for now
3) replace BundleReport to use the SharedBundleReport to avoid disconnect bw data here and data from shared

(could you make an issue capturing this when you get a chance please?)

Reply:
https://github.com/codecov/engineering-team/issues/1067
https://github.com/codecov/engineering-team/issues/1068

Yep will refactor this code next when tackling these two tickets.